test/www/jxcore/bv_tests/testThaliMobileNative.js

```diff
@@ -1,5 +1,14 @@
 'use strict';
 
+// Issue #419
+var ThaliMobile = require('thali/NextGeneration/thaliMoblie');
+if (global.NETWORK_TYPE === ThaliMobile.networkTypes.NATIVE ||
+    global.NETWORK_TYPE === ThaliMobile.networkTypes.BOTH) {
+  // keep going
+} else {
```

Comment:
Will this switch be extended with `else if` in future? We can use `if ( a !== b ) { return }` if not.